### DISCUSSION TITLE
Document Internationalization

### DISCUSSION THREAD

**Discussion by lecepin** [pmndrs/jotai#1633] [General]
hi @dai-shi ~

Are there any plans to support multiple languages in jotai's documentation?
For example Chinese.

If there is a plan for this, I can contribute.

**Comment by dai-shi** [comment]
At this point, we have no plans to support multiple languages.
What we can try is to do it outside of the official repo.
One naive idea is to fork the repo and translate docs on your end.
I've done this way in one of my projects before.
